src/routes/+page.svelte

```diff
@@ -45,13 +43,19 @@
             <Forms />
             <Mobile />
             <Keyboard />
+
+            <ThemeSwitcher class="absolute top-[1em] right-[1em]" />
         </div>
 
         <hr />
 
         <A11yListView />
     </div>
-    <div slot="second" class="w-full h-full flex justify-center items-center">
-        <A11yInspector />
+    <div slot="second" class="flex flex-col h-full w-full items-stretch justify-center">
+        <div class="flex-grow">
+            <A11yInspector />
+        </div>
+        <div class="flex-grow-0 text-right">
```

Comment:
Ein leeres Element mit flex=0 wirkt nicht so als ob es einen Nutzen hat. Warum genau wird das benötigt?

Reply:
falls man was untendrunter packen möchte. war aber eigentlich nur ein überbleibsel, das ich drin gelassen habe damit da im zweifel was rein kann. ich hab immer gern n paar vorbereitete, angedachte stellen, in die man im zweifel direkt was packen kann. löschen kann man ja immer wenn es stört.
kann aber natürlich auch raus. ich dachte mir nur "haben oder nicht haben"